### ISSUE TITLE
#383 breaks VoiceOver selection speech

### ISSUE THREAD

**Issue by jhersh** [TTTAttributedLabel/TTTAttributedLabel#397]
#383 is super neat, however the change to return `NO` from `isAccessibilityElement` means that VoiceOver will no longer speak the label's text when it is selected.

I did not adequately test accessibility before my most recent release and this change has rendered my app completely unusable for VoiceOver users. I reverted to 1.9.5.

**Comment by PatrickNLT** [comment]
I confirm that VoiceOver doesn't work when the label is added to a table view cell. Tested on iOS 9.1.
I created a demo project: https://github.com/PatrickNLT/TTTAttributedLabelAccessibilityTest

Apparently Apple is doing some pretty hacky stuff with the accessibility in table views (see http://lists.apple.com/archives/accessibility-dev/2013/Oct/msg00008.html for instance). Maybe it's related.
